engine/apps/slack/tasks.py

```diff
@@ -303,14 +303,14 @@ def post_slack_rate_limit_message(integration_id):
         return
 
     default_route = integration.channel_filters.get(is_default=True)
-    if (slack_channel_id := default_route.slack_channel_id_or_org_default_id) is not None:
+    if (slack_channel := default_route.slack_channel_or_org_default) is not None:
```

Comment:
related to [this change](https://github.com/grafana/oncall/pull/5314/files#diff-5b78eae626bcf20076ff2f5f25daa3f5703ff8dd9336e5d3e3dbbd7aa827d6b2R175)